### PR TITLE
156: Create reusable .nib to consolidate large black buttons across the app ("next", "save", "checkout")

### PR DESCRIPTION
**Pull request by MiteshiOS**
## Notes

- Added reusable nib for the large black button
- Implemented in product details for Add To Bag & Buy It Now

## Visuals
Screenshot 2022-08-05 at 8 34 35 PM
Screenshot 2022-08-05 at 8 36 15 PM

### CODE REVIEW

File: archiv333/View Controllers/Home/ProductViewController.swift [hunk 1]
```diff
@@ -21,12 +21,12 @@ class ProductViewController: UIViewController, ColorSelectionDelegate, ActionsTr
...
     @IBOutlet var vvFloatingTrayView: UIView!
-    @IBOutlet var vvAddToBag: UIView!
-    @IBOutlet var btAddToBag: SpacingButton!
-    @IBOutlet var ivAddToBagIcon: UIImageView!
-    @IBOutlet var vvBuyItNow: UIView!
-    @IBOutlet var btBuyItNow: SpacingButton!
-    @IBOutlet var ivBuyItNowArrow: UIImageView!
+    @IBOutlet var vvAddToBag: LargeBlackButton!
+    //@IBOutlet var btAddToBag: SpacingButton!
+    //@IBOutlet var ivAddToBagIcon: UIImageView!
```
Comment (tejen) [inline archiv333/View Controllers/Home/ProductViewController.swift]: 🧹 Please cleanup all code in this PR, including this unused code. @MiteshiOS

**Comment by MiteshiOS** [comment]
[Create reusable .nib to consolidate large black buttons across the app ("next", "save", "checkout")](https://trello.com/c/GcLfSZSB/156-create-reusable-nib-to-consolidate-large-black-buttons-across-the-app-next-save-checkout)

File: archiv333/View Controllers/Authentication/SignupEmailViewController.swift [hunk 2]
```diff
@@ -86,20 +87,22 @@ class SignupEmailViewController: MovesWithKeyboardViewController {
         navigationController!.popToRootViewController(animated: true)
     }
 
-    @IBAction func didTapNext(_ sender: Any) {
-        view.endEditing(true)
-        
-        tfInput.text = tfInput.text?.trimmingCharacters(in: .whitespacesAndNewlines)
```
Comment (tejen) [inline archiv333/View Controllers/Authentication/SignupEmailViewController.swift:92]: 🧹 Please update nomenclature to discontinue inappropriate usage of `didTap` notation @MiteshiOS

**Review by tejen — CHANGES_REQUESTED** [review]

File: archiv333/View Controllers/Authentication/SignupEmailViewController.swift (hunk 2 above)
Comment (MiteshiOS) [reply archiv333/View Controllers/Authentication/SignupEmailViewController.swift:92]: Done @tejen

**Review by tejen — APPROVED** [review]

**Pull request merged**